packages/openwrt/net/nDPId-testing/Makefile

```diff
@@ -73,7 +82,14 @@ CMAKE_OPTIONS += -DSTATIC_LIBNDPI_INSTALLDIR="$(PKG_BUILD_DIR)/libnDPI/install"
 TARGET_CFLAGS += -DLIBNDPI_STATIC=1
 TARGET_CFLAGS += -Werror
 
-ifneq ($(CONFIG_NDPI_TESTING_INFLUXDB),)
+ifneq ($(CONFIG_NDPID_TESTING_PFRING),)
+CMAKE_OPTIONS += -DENABLE_PFRING=ON \
+	-DPFRING_KERNEL_INC="$(KERNEL_BUILD_DIR)/PF_RING-8.4.0/kernel" \
```

Comment:
hard-coding `PF_RING-8.4.0` is a bad idea, but because of PF_RING which does not install it's kernel module header file, it's necessary until another solution is found